docs/conceptual/gpu-arch.md

```diff
@@ -22,11 +22,19 @@ accelerators and the CDNA™ 2 architecture that is the foundation of these GPUs
 Review hardware aspects of the AMD Instinct™ MI100
 accelerators and the CDNA™ 1 architecture that is the foundation of these GPUs.
 
-* [Instruction Set Architecture](https://www.amd.com/system/files/TechDocs/instinct-mi100-cdna1-shader-instruction-set-architecture%C2%A0.pdf)
+* [Instruction set architecture](https://www.amd.com/system/files/TechDocs/instinct-mi100-cdna1-shader-instruction-set-architecture%C2%A0.pdf)
```

Comment:
This is abbreviated as ISA, may add ISA?

Reply:
Instruction Set Architecture is a type of document, from what I know. It's usually abbreviated to ISA. Given the abbreviation, we should probably keep the uppercase.

Reply:
Sure; made the changes.